nntrainer/src/conv2d_layer.cpp

```diff
@@ -111,7 +114,17 @@ int Conv2DLayer::setOptimizer(Optimizer &opt) {
   if (status != ML_ERROR_NONE)
     return status;
 
-  return this->opt.initialize(dim, true);
+  std::vector<Tensor> list_d;
+  for (unsigned int i = 0; i < filter_size; ++i) {
+    for (unsigned int j = 0; j < 2; ++j) {
```

Comment:
Does this loop of 2 creates `wm, wv` pair? If yes, lets do this in the optimizer than here.

Reply:
Keep it for now. when we fix the weights issue, then we can replace it.